openedx_webhooks/tasks/pr_tracking.py

```diff
@@ -263,6 +263,7 @@ def desired_support_state(pr: PrDict) -> Optional[PrDesiredInfo]:
     if not has_signed_agreement:
         desired.bot_comments.add(BotComment.NEED_CLA)
         desired.jira_initial_status = "Community Manager Review"
+        desired.github_labels.add('NEED-CLA')
```

Comment:
Naming convention is fairly arbitrary. I went with:
- ALL CAPS: to make it stand out
- hyphen instead of underscore: following convention used above